### PR TITLE
Update Payments docs with missing information

### PR DESCRIPTION
**Pull request by eliasturbay**
# Why

The docs to integrate the Stripe SDK on Android are outdated.

# How

Added just a couple of missing steps in the process.

# Test Plan

N/A

### CODE REVIEW

File: docs/versions/unversioned/sdk/payments.md [hunk 1]
```diff
@@ -57,6 +57,11 @@ _Note_: These steps are required only if you have ejected your app with SDK < 30
 1.  Add these lines into your settings.gradle file.
 
 ```groovy
+// make sure this two lines are there
+include ':expo-core'
+project(':expo-core').projectDir = new File(rootProject.projectDir, '../node_modules/expo-core/android')
+
+// add this two lines
```
Comment (brentvatne) [inline docs/versions/unversioned/sdk/payments.md]: I don't think this comment is necessary

```suggestion
```

File: docs/versions/unversioned/sdk/payments.md [hunk 2]
```diff
@@ -57,6 +57,11 @@ _Note_: These steps are required only if you have ejected your app with SDK < 30
 1.  Add these lines into your settings.gradle file.
 
 ```groovy
+// make sure this two lines are there
```
Comment (brentvatne) [inline docs/versions/unversioned/sdk/payments.md]: I don't think this comment is necessary

```suggestion
```

File: docs/versions/unversioned/sdk/payments.md [hunk 3]
```diff
@@ -102,6 +107,19 @@ implementation project(':expo-payments-stripe')
     }
 ```
 
+6.  Add `StripePackage` to the list of Expo Packages in `your-project/android/app/src/main/java/host/exp/exponent/MainApplication.java`.
```
Comment (brentvatne) [inline docs/versions/unversioned/sdk/payments.md]:
```suggestion
6.  Add `StripePackage` to the list of Expo Packages in `android/app/src/main/java/host/exp/exponent/MainApplication.java`.
```

File: docs/versions/unversioned/sdk/payments.md [hunk 4]
```diff
@@ -102,6 +107,19 @@ implementation project(':expo-payments-stripe')
...
+
+```groovy
+  public List<Package> getExpoPackages() {
+    return Arrays.<Package>asList(
+        ...
```
Comment (brentvatne) [inline docs/versions/unversioned/sdk/payments.md]:
```suggestion
        // other packages here
```

File: docs/versions/unversioned/sdk/payments.md [hunk 5]
```diff
@@ -102,6 +107,19 @@ implementation project(':expo-payments-stripe')
...
+        new BackgroundFetchPackage(),
```
Comment (brentvatne) [inline docs/versions/unversioned/sdk/payments.md]:
```suggestion
```

**Review by brentvatne — CHANGES_REQUESTED** [review]
left some suggested changes, thanks!

**Comment by nicknovitski** [comment]
The docs pages have moved from `docs/versions` to `docs/pages/versions`.  Rebasing should resolve the conflicts and fix the client build.

**Comment by eliasturbay** [comment]
Excuse me @nicknovitski, rebasing against which branch? Thx!

**Comment by esamelson** [comment]
Hi @eliasturbay - apologies for the slow followup here, if you could rebase again current expo master that would be great 😄

**Comment by springcoil** [comment]
@eliasturbay https://stackoverflow.com/questions/7929369/how-to-rebase-local-branch-with-remote-master/7929499 if you're having any difficulty with your rebasing. (I know I've had a lot of trouble in the past with OSS work :) so I hope this helps.)

**Comment by sjchmiela** [comment]
These instructions are outdated now since we introduced autolinking of unimodules. Sorry it took us so long to look into this PR and thank you for your contribution! ❤️

**Pull request closed without merging**